turbomoleio/core/molecule.py

```diff
@@ -69,7 +88,71 @@ def get_mol_and_indices_frozen(string):
         if len(s) == 5 and s[-1].lower() == "f":
             frozen_indices.add(iatom)
 
-    mol = Molecule(species, coords)
+    coords = np.array(coords)
+    if cell_string is not None:
+        if lattice_string is not None:
+            raise ValueError('Only cell_string or lattice_string should be set.')
+        periodicity = int(periodic_string.strip())
```

Comment:
In principle, it will not be None for the call in PeriodicSystem.from_string and there is a check in that method. Do you think we should add another check here ?

Reply:
it depends if we expect this to be called somewhere else. If not, it can stay this way.  
In principle the check could be moved from the `from_string` method to here, but if the error is raised it will be much less clear for a user what is the real problem with the input file (i.e. a datagroup is missing).